.github/workflows/tests.yml

```
@@ -72,4 +72,9 @@ jobs:
 
     - name: Tests
       run: |
-        pytest tests --ignore tests/integration_tests
+        if [ ${{ matrix.python-version }} = 3.9 ]; then
+          pytest tests --ignore tests/integration_tests \
+            --deselect tests/samplers_tests/test_samplers.py::test_nan_objective_value --deselect tests/samplers_tests/test_samplers.py::test_partial_fixed_sampling
+        else  
```

Please remove trailing whitespaces.
```suggestion
        else
```

---

Thanks for the review. Changed in the latest commit 👍